atcoder/maxflow.py

```diff
@@ -0,0 +1,209 @@
+from typing import NamedTuple, Optional, List
+
+
+class MaxFlow:
+    class Edge(NamedTuple):
+        src: int
+        dst: int
+        cap: int
+        flow: int
```

Comment:
Used `src, dst` instead of `from, to` because `from` is a reserved word in python.